### PR TITLE
SFTP in queue architecture

### PR DESCRIPTION
**Pull request by mry-usds**
This PR does adds support for SFTP via the new Queue architecture

## Changes
- Triggers from a message in the queue vs the blobstore
- pulls parameters from the env
- added SFTP_SETUP.md to give guidance on setting up local SFTP

## Checklist
- [X] Tested locally?
- [X] Added new dependencies?
- [X] Updated the docs?
- [ ] Added a test?
- [X] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## Fixes 

- #issue

## To Be Done
- add support to get users/passwords into env in an easier manner

### CODE REVIEW

File: prime-router/SFTP-SETUP.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Setting up SFTP
```
Comment (RickHawesUSDS) [inline prime-router/SFTP-SETUP.md:1]: +1 For creating a doc. Can you reference this doc in the getting started.md doc?

File: prime-router/docker-compose.yml [hunk 2]
```diff
@@ -25,4 +25,13 @@ services:
...
-      - 10001:10001
\ No newline at end of file
+      - 10001:10001
+
+#local SFTP server as a receive point
+  sftp:
+    image: atmoz/sftp
+    volumes:
+        - ./target/sftp:/home/foo/upload
+    ports:
```
Comment (RickHawesUSDS) [inline prime-router/docker-compose.yml:35]: Thinking about it. I like that you are mapping the a target folder. It gets cleaned with `mvn clean`

File: prime-router/src/main/kotlin/azure/SendFunction.kt [hunk 3]
```diff
@@ -32,19 +32,24 @@ class SendFunction {
...
+                error( "Unable to find a service handler for ${header.destination}")
 
-            //context.logger.info( "Writing to ${mockServer.getBaseDirectory().toString()}" )
-            //val session = initSshClient()
-            //val sendKlass = Class.forName("gov.cdc.prime.router.SftpSend").kotlin
+            context.logger.info( "Transport found for ${service.fullName} = ${service.transport.type}")
 
-            val transportMetadata: OrganizationService.Transport = lookupTransportMetadata()
-            val transport = SftpTransport() // TODO:  look up the correct class to call based on the transport metadata
+            var transportSuccessful = when( service.transport.type ){
```
Comment (RickHawesUSDS) [inline prime-router/src/main/kotlin/azure/SendFunction.kt:40]: Just noticed that you are using a non-standard spacing. I'd like to see us standardized on the Kotlin style guide recommendations.

**Review by RickHawesUSDS — APPROVED** [review]
Looks good. Great to see.

File: prime-router/src/main/kotlin/azure/SendFunction.kt (hunk 3 above)
Comment (mry-usds) [reply prime-router/src/main/kotlin/azure/SendFunction.kt:40]: I just saw a linter for kotlin - I'll see if I can add that in...

**Pull request merged**